src/main/java/org/killbill/billing/entitlement/api/Entitlement.java

```diff
@@ -372,4 +378,13 @@ public Entitlement changePlanOverrideBillingPolicy(final EntitlementSpecifier sp
     @RequiresPermissions(ENTITLEMENT_CAN_CREATE)
     public void updateBCD(final int bcd, final LocalDate effectiveFromDate, final CallContext context) throws EntitlementApiException;
 
+    /**
+     * @param quantity          the quantity or equivalent number of subscriptions that should be billed
+     * @param effectiveFromDate date after which that BCD change becomes active
```

Comment:
Typo in Javadoc (BCD)

Reply:
👍 Fixed in https://github.com/killbill/killbill-api/pull/112/commits/45bd6cf1e03af8b746c9fc15a4d8677235ac2db4